### PR TITLE
Bitcoin testnet

### PR DESCRIPTION
**Pull request by dolgopolovwork**
Moving bitcoin to test network. Also now we have two spring profiles: local and testnet. Switching between profiles is based on environment variable PROFILE.

### CODE REVIEW

File: btc-deposit/src/main/kotlin/notary/btc/BtcNotaryInitialization.kt [hunk 1]
```diff
@@ -55,36 +56,46 @@ class BtcNotaryInitialization(
...
     }
 
+    /**
+     * Starts bitcoin blockchain downloading process
+     */
+    private fun startChainDownload(wallet: Wallet) {
+        logger.info { "Start bitcoin blockchain download" }
+        val networkParams = btcNetworkConfigProvider.getConfig()
+        val levelDbFolder = File(btcNotaryConfig.bitcoin.blockStoragePath)
+        // TODO - D3-321 - dolgopolov.work - I can see nasty logs here. Try to fix it
```
Comment (grimadas) [inline btc-deposit/src/main/kotlin/notary/btc/BtcNotaryInitialization.kt]: Remove this

File: build.gradle [hunk 2]
```diff
@@ -250,7 +250,7 @@ def getEthPasswordsArgs() {
     return args
 }
 
-task retrieveIrohaBindings(type: Exec) {
+task retrieveIrohaBindings(type: Exec){
```
Comment (Alexey-N-Chernyshov) [inline build.gradle]: Spaces before braces are inconsistent.

File: configs/btc/registration_testnet.properties [hunk 3]
```diff
@@ -1,16 +1,17 @@
 # ========= Bitcoin Registration Service =========
 btc-registration.registrationAccount=btc_registration_service_red@notary
 btc-registration.mstRegistrationAccount=mst_btc_registration_service_red@notary
+btc-registration.notaryAccount=notary_red@notary
 # port of registration service
 btc-registration.port=8084
-# --------- Credetnials ------
-btc-registration.registrationCredential.accountId=notary_red@notary
+# ---------- Credentials ---------
+btc-registration.registrationCredential.accountId=btc_registration_service_red@notary
```
Comment (Alexey-N-Chernyshov) [inline configs/btc/registration_testnet.properties]: please, add keypair for btc_registration_service_red

Comment (dolgopolovwork) [reply configs/btc/registration_testnet.properties]: this will be implemented in separate task

**Pull request merged**